test/e2e/test-apps/other/browser-tracing/event.json

```diff
@@ -112,5 +112,8 @@
       "event.environment": "javascript",
       "event.origin": "electron"
     }
+  },
+  "dynamicSamplingContext": {
+    "sample_rate": "1"
```

Comment:
Here we modify the browser tracing e2e test to ensure the `sample_rate` is indeed in the DSC